docs/release-notes/Percona-Everest-1.6.0-(2025-03-31).md

```diff
@@ -0,0 +1,127 @@
+
+    - **Resource quota check** - Verify that your resource quotas allow for the requested storage capacity. See the [Kubernetes documentation on Storage Resource Quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/#storage-resource-quota){:target="_blank"} for more information.
+
+    **How to modify storage capacity**
+
+    To increase the DISK for a database, go to the Percona Everest home page and select your desired database. Then, navigate to **Overview > Resources > Edit** and enter the new value for **DISK** (in Gi).
+
+    ![!image](../images/edit_storage_capacity.png)
+
+
+=== "Major database upgrades"
+
+    ### Seamless major version upgrades for MongoDB
+
+    Percona Everest 1.6.0 introduces support for major version upgrades of MongoDB, enabling you to upgrade your databases with **minimal downtime and disruption**. This enhancement ensures your applications remain secure, performant, and compliant with the latest MongoDB features.
+
+    To upgrade your MongoDB databases, visit the Percona Everest homepage and select the database you want to upgrade. Go to the **Overview** page, click on **DB Details > Edit**, choose the desired version, and then click **Upgrade**.
+
+    ![!image](../images/database_upgrade.png)
+    
+    ![!image](../images/upgrade_database_versions.png)
+
+
+
+=== "Operators support"
+
+    ### Support for Percona Operator for PostgreSQL 2.6.0
+
+    Percona Everest 1.6.0 includes support for Percona Operator for PostgreSQL 2.6.0.
+
+
+## Google Container Registry (GCR)
+
+!!! warning "GCR deprecation"
+    GCR is set to be **deprecated**, with its official shutdown scheduled for **May 20, 2025**.
+
+    All Percona Everest versions prior to 1.4.0 depend on images hosted on Google Container Registry (GCR). These images will become unavailable after the shutdown date: **May 20, 2025**.
+
+### Impact of GCR deprecation
+
+Percona Everest versions older than 1.4.0 will cease to function after this date.
+
+### Action required
+
+We strongly recommend upgrading to Percona Everest version 1.4.0 or later as soon as possible. If you do not upgrade, Percona Everest will no longer function.
+    
+For more details, refer to the [Container Registry Deprecation documentation](https://cloud.google.com/artifact-registry/docs/transition/prepare-gcr-shutdown){:target="_blank"}.
+
+
+## New features
+
+- [EVEREST-1870](https://perconadev.atlassian.net/browse/EVEREST-1870), [EVEREST-1871](https://perconadev.atlassian.net/browse/EVEREST-1871), [EVEREST-1872](https://perconadev.atlassian.net/browse/EVEREST-1872): Starting with Percona Everest 1.6.0, you can leverage manual storage scaling to increase the capacity of your **S3-compatible storage** for MySQL, MongoDB, and PostgreSQL databases respectively.
```

Comment:
Same deal as https://github.com/percona/everest-doc/pull/227#discussion_r2026230847, this is PVC storage not S3

Reply:
Fixed.